css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,77 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "DM Sans", sans-serif;
+  font-weight: 700;
+}
+
+.container {
+  max-width: 900px;
+  margin: 0 auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.col-fifth {
+  flex-basis: calc(100% / 5);
+}
+
+.col-full {
+  flex-basis: 100%;
+}
+
+h1 {
+  text-align: center;
+  font-size: 24px;
+  margin-bottom: -10px;
+}
+
+img {
+  width: 100%;
+  height: auto;
```

Comment:
I would just give this a set height, and use `object-fit` on the images to adjust how they render in the available space.